index.html

```diff
@@ -105,7 +119,14 @@ <h1 class="blog-title">Read Our Latest Posts. Learn Something New Maybe!</h1>
                         <div class="post">
                             <div class="post-imgWrapper">    
```

Comment:
`post-imgWrapper` -> `post-img-wrapper`
Please fix all similar issues in the document.